frontend/src/store/modules/taste/state.js

```diff
@@ -1,5 +1,4 @@
 export default {
     allTastes: [],
```

Comment:
you use allTastes as an object everywhere with a property "byId", so create set here default shape